VideoLocker/res/layout/panel_find_course.xml

```diff
@@ -36,7 +36,7 @@
             android:layout_below="@+id/findcourse_tv"
             android:layout_centerInParent="true"
 			android:padding="10dp"
-            android:background="@drawable/bt_blue"
+            android:background="@drawable/edx_creation_button"
```

Comment:
This should be renamed since it's not really a creation button here. `edx_blue_button`?